dbt/adapters/spark/impl.py

```diff
@@ -60,6 +61,9 @@ class SparkAdapter(SQLAdapter):
         'stats:rows:description',
         'stats:rows:include',
     )
+    INFORMATION_COLUMNS_REGEX = re.compile(
+        r"\|-- (.*): (.*) \(nullable = (.*)\b", re.MULTILINE)
+    INFORMATION_OWNER_REGEX = re.compile(r"^Owner: (.*)$", re.MULTILINE)
```

Comment:
While we're here... any chance I could convince you to regex for stats as well? :)
```python
INFORMATION_STATISTICS_REGEX = re.compile('^Statistics: (.*)$", re.MULTILINE)
```

From some (anecdotal) testing, delta tables include a line like `Statistics: 1109049927 bytes`, whereas (e.g.) parquet tables include a line like `Statistics: 1109049927 bytes, 14093476 rows`.

The `SparkColumn` object takes a [`table_stats`](https://github.com/fishtown-analytics/dbt-spark/blob/40db36435f8938b5c45dc0f2277e7fa504ea8b6c/dbt/adapters/spark/column.py#L18) argument; we'd just need to adjust [`convert_table_stats`](https://github.com/fishtown-analytics/dbt-spark/blob/40db36435f8938b5c45dc0f2277e7fa504ea8b6c/dbt/adapters/spark/column.py#L43-L57) to handle the delta table case, which is missing the `rows` bit after the comma split.

Alternatively, if you wanted to do it all in regex, I guess you could:
```python
INFORMATION_STATISTICS_REGEX = re.compile('^Statistics: ([0-9]+) bytes(, ([0-9]+) rows)?', re.MULTILINE)
```
```
bytes, _, rows = re.findall(self.INFORMATION_OWNER_REGEX, relation.information)
```

Reply:
@jtcohen6 This is a great idea! This is useful information and I think it's worth trying to parse it. I'll give it a try and add an extra test for parquet tables. I'll let you know when I have this implemented. (Switching back to WIP)